specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/certificates.json

```diff
@@ -1331,6 +1331,11 @@
           "type": "string",
           "format": "base64url",
           "description": "Thumbprint of the certificate."
+        },
+        "entityId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The system-generated ID for this secret."
```

Comment:
Seems to be a copy/paste error: you mention "secret" here for certificates. Same for keys. Please be sure to update them all.